### ISSUE TITLE
Bug Found in  Accord.Math.Distance.Levenshtein

### ISSUE THREAD

**Issue by LSGOMYP** [accord-net/framework#232]
I found a bug in  Accord.Math.Distance.Levenshtein

the if statement

```
        public static double Levenshtein(string x, string y)
        {
            if (x == null || x.Length == 0)
            {
                if (y == null || y.Length != 0)
                    return 0;
                return y.Length;
            }
            else
            {
                if (y == null || y.Length == 0)
                    return x.Length;
            }
```
in my opinion that the if statement may be like the fowllowing statement
```
       public static double Levenshtein(string x, string y)
        {
            if (x == null || x.Length == 0)
            {
                if (y == null || y.Length == 0)
                    return 0;
                return y.Length;
            }
            else
            {
                if (y == null || y.Length == 0)
                    return x.Length;
            }
```

**Comment by zgrkpnr** [comment]
This is not a bug. You are making a suggestion. The above code is perfectly fine.

**Comment by cesarsouza** [comment]
You are perfectly right! Many thanks for reporting the issue.

**Comment by cesarsouza** [comment]
Fixed on release 3.2.
